optimisation/ScanSelectionsTree.py

```diff
@@ -75,6 +77,16 @@
         upperLowerCuts.append('>')
     varNames.append(var)
 
+
+# load DCA value to scan
+dcaMins = inputCfg['DCA']['DCAmin']
```

Comment:
Could you add an if enableDCA here so that if it is false one does not have to define the DCA bins?